src/pydap/server/devel_ssl.py

```diff
@@ -44,36 +45,36 @@ class LocalTestServerSSL(LocalTestServer):
     Relies on multiprocessing and is usually slow (it has to
     start and shutdown which typically takes ~2 sec).
 
-    Usage:
-    >>> import numpy as np
-    >>> from pydap.handlers.lib import BaseHandler
-    >>> from pydap.model import DatasetType, BaseType
-    >>> DefaultDataset = DatasetType("Default")
-    >>> DefaultDataset["byte"] = BaseType("byte", np.arange(5, dtype="B"))
-    >>> DefaultDataset["string"] = BaseType("string", np.array(["one", "two"]))
-    >>> DefaultDataset["short"] = BaseType("short", np.array(1, dtype="h"))
-    >>> DefaultDataset
-    <DatasetType with children 'byte', 'string', 'short'>
+    # Usage:
```

Comment:
ditto

Reply:
I commented this because it was producing some errors in my local testing environment. That was on an earlier version of the PR. I completely forgot about these comments.

Reply:
uncommented. test continue to pass. I will mark this as resolved